dashboard/db/migrate/20160429214539_add_script_to_course_units.rb

```diff
@@ -0,0 +1,6 @@
+class AddScriptToCourseUnits < ActiveRecord::Migration
+  def change
+    Plc::CourseUnit.destroy_all
```

Comment:
I don't think it's safe to destroy PLC objects anymore - I think there have been some created.

Can you make the field not required? We can fix the association at some point, and then make another migration to make the column required.

It also might be worth separating this into an up and down migration. In the down migration, through some "cannot rollback" exception. That way, if someone accidentally does a rake db:Migrate:redo on production, they'll get blocked.

Similar comment for the other migrations